src/App.js

```diff
@@ -1,5 +1,90 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
 class App {
-  async play() {}
+
+  getRandomNumber(){
+    const computerNumber = [];
```

Comment:
저는 변수명만 보고 의도, 타입을 파악하기 쉽게 아래와 같이 작성하는 편인데 어떻게 생각하시나요??

```suggestion
    const computerNumberList = [];
```

Reply:
와! 이렇게 변수명을 쓰면 타입같은 것도 확실히 잘 전달되겠네요! 좋은 것 같습니다.